unified RBAC: add EM in v1.0 (#16354)
microsoftgraph/microsoft-graph-docs: api-reference/v1.0/api/rbacapplication-post-roleassignments.md

## Permissions

> 
> Depending on the RBAC provider and the permission type (delegated or application) that is needed, choose from the following table the least privileged permission required to call this API. To learn more, including [taking caution](/graph/auth/auth-concepts#best-practices-for-requesting-permissions) before choosing more privileged permissions, search for the following permissions in [Permissions](/graph/permissions-reference).
> 
> ### For the Directory (Azure AD) provider

Reviewer: Same comment
```suggestion

One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

### For the directory (Azure AD) provider
```

}
```

> ### Example 4: Create a role assignment at access package catalog scope

Reviewer: ```suggestion
### Example 4: Create a role assignment with access package catalog scope
```
